<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_dataset_widget</class>
 <widget class="QWidget" name="new_dataset_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="central_alf_atom_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="central_atom_settings_groupbox">
        <property name="title">
         <string>Atom Center and Property Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="4">
          <widget class="QLabel" name="xy_plane_atom_label">
           <property name="text">
            <string>XY-plane Atom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="atom_names_combo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="central_alf_atom_label">
           <property name="text">
            <string>Central Atom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="x_axis_atom_label">
           <property name="text">
            <string>X-axis Atom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="x_axis_atom_combo_box"/>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="xy_plane_atom_combo_box"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="properties_cmap_combo_box"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="current_property_textbox">
           <property name="text">
            <string>Property</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="calculate_geometries_button">
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="plotting_settings_groupbox">
        <property name="title">
         <string> Plotting Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0" rowspan="2">
          <widget class="QGroupBox" name="individual_point_settings_groupbox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Point Settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0">
             <widget class="QWidget" name="individual_points_widget" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="property_value_for_current_point_label">
                 <property name="text">
                  <string>Property Value for this Point: </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QSlider" name="individual_point_slider">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="individual_point_box">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="property_value_for_current_point">
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Data Has Not been read in yet</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="plot_all_points_radio">
              <property name="text">
               <string>Plot All Points</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">point_settings_button_group</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="plot_individual_point_radio">
              <property name="text">
               <string>Plot Individual Points</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">point_settings_button_group</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Coloring Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="random_colors_radio">
              <property name="text">
               <string>Random Colors</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">color_settings_button_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="default_atom_colors_radio">
              <property name="text">
               <string>Default Atom Colors</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">color_settings_button_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="central_atom_color_for_all_radio">
              <property name="text">
               <string>Central Atom Color for All</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">color_settings_button_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="cmap_radio">
              <property name="text">
               <string>Cmap for Selected Property</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">color_settings_button_group</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="atoms_to_plot_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_26">
      <item row="2" column="0">
       <widget class="QWidget" name="plotted_atoms_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="atoms_to_plot_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Plotted Atoms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="atoms_to_plot_scroll_area">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="atoms_to_plot_grid">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>981</width>
              <height>213</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_27"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="how_hide_all_atoms_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="show_all_plotted_atoms_button">
           <property name="text">
            <string>Show All Atoms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hide_all_plotted_atoms_button">
           <property name="text">
            <string>Hide All Atoms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="atom_colors_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QLabel" name="atom_colors_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Atom Colors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="atom_color_scroll_area">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="atom_color_scroll_area_grid">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>981</width>
              <height>213</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_28"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="point_settings_button_group"/>
  <buttongroup name="color_settings_button_group"/>
 </buttongroups>
</ui>
